tests/test_visualization.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+from aopy.visualization import *
+import numpy as np
+
+write_dir = 'tests/tmp'
```

Comment:
not sure which is better - absolute path or relative path. 

I would add something like 
ROOT_DIR = os.path.abspath(os.curdir)
write_dir = os.path.join(ROOT_DIR, "tests/tmp")